<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWidget</class>
 <widget class="QWidget" name="PlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treePlot">
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>临时目录（固态）</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最终目录（机械）</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>当前数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>开始时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用时</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>使用内存</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>进度</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="checkBoxPhase1Limit">
       <property name="text">
        <string>限制阶段1的任务数量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxPhase1Count">
       <property name="suffix">
        <string>个</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTotalLimit">
       <property name="text">
        <string>限制总任务数量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxTotalCount">
       <property name="suffix">
        <string>个</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAutoRestartMine">
       <property name="text">
        <string>完成后自动重新挖矿</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCreatePlot">
       <property name="text">
        <string>创建任务</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
